bundler/helpers/v2/lib/functions.rb

```diff
@@ -143,7 +143,7 @@ def self.set_bundler_flags_and_credentials(dir:, credentials:)
 
       Bundler.settings.set_command_option(
         cred.fetch("host"),
-        token.gsub("@", "%40F").gsub("?", "%3F")
+        token.gsub("@", "%40").gsub("?", "%3F")
```

Comment:
Is this related to the fix?

Reply:
Only in that testing private gem server access via dry-run.rb was broken because of that spurious F.

Reply:
I see, so this bug does not affect end users at all, only `dry-run.rb`, correct?

Reply:
Yes.